libr/util/d/Makefile

```diff
@@ -0,0 +1,30 @@
+include ../../../global.mk
+
+SDBCHARSETPATH=$(DESTDIR)$(DATADIR)/radare2/$(VERSION)/util
+CWD=$(shell pwd)
+
+all:
```

Comment:
this ishould be done for meson too. maybe @thestr4ng3r or @ret2libc can give you some advices on this

Reply:
ping @thestr4ng3r @ret2libc

Reply:
I think you should look at `libr/anal/d/meson.build`.

Reply:
I am looking how to list files in the current directory with meson.

Reply:
As said, just look at that file and there should be everything.